### ISSUE TITLE
matlab implemention of loam

### ISSUE THREAD

**Issue by jeannotes** [CMobley7/Loam_Matlab_SMP#1]
Hi is this matlab implemention of loam, I find it is still c++ source code.

**Comment by CMobley7** [comment]
No, I believe we were using MATLAB to serve up the images at the time. I don't believe a MATLAB implementation of LOAM exists. I suppose you could always translate it from C++ to MATLAB, but it would probably be pretty slow. So, I suggest just using the C++ implementation with ROS and using python as the glue to whatever other packages you need.
